search_that_hash/__main__.py

```diff
@@ -69,6 +69,22 @@ def main(**kwargs):
             sth --text "5f4dcc3b5aa765d61d8327deb882cf99"
     """
 
+    if kwargs == {  # pragma: no cover
```

Comment:
Don't ignore this, you can 100% test whether or not the program has no input lol

Reply:
Done :)